### PR TITLE
Dotnet OS agent headers

### PR DESCRIPTION
**Pull request by sacOO7**
- Added OS specific agent headers for dotnet

### CODE REVIEW

File: protocol/agents.json [hunk 1]
```diff
@@ -155,6 +155,61 @@
       "versioned": false,
       "type": "runtime"
     },
+    {
+      "identifier": "xamarin",
+      "versioned": false,
+      "type": "runtime"
+    },
+    {
+      "identifier": "dotnet6",
```
[the remark this answers is not in the document]
Comment (sacOO7) [reply protocol/agents.json:164]: Last time I checked, that API wasn't much stable. I will recheck and update 👍

[the remark this answers is not in the document]
Comment (sacOO7) [reply protocol/agents.json:164]: This has been updated as per your suggestion!

**Review by owenpearson — APPROVED** [review]
LGTM, thanks

**Comment by sacOO7** [comment]
Thanks @owenpearson : )

**Pull request merged**